### PR TITLE
add windows build

### PR DESCRIPTION
**Pull request by grlee77**
This PR is to test enabling a Windows build. With the 0.11 release, it is possible to build with only a subset of precisions present (e.g. missing long double on the conda-forge windows FFTW package).

I did not have access to upload pyFFTW to PyPI, so for now this is using the github tag. I can change that back once the packages are on PyPI.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
Would add `git` to the `build` deps.

**Comment by grlee77** [comment]
I first tried static linking, but that gave some errors about unresolved symbols during linking:
https://ci.appveyor.com/project/conda-forge/pyfftw-feedstock/builds/20192726/job/qk2yx8oq2c7v6c3i#L678

A build in the most recent commit using the `.dlls` succeeded, but import of the package failed during the testing phase:
https://ci.appveyor.com/project/conda-forge/pyfftw-feedstock/builds/20192792/job/hteyl00kpux9tdpw#L911

I know I had successfully built this on a windows machine (not via CI) some months back, but I couldn't find  the recipe I used at that time.

**Comment by grlee77** [comment]
For the .dll case, I guess I probably just need `fftw` in the run section of meta.yaml.

**Comment by grlee77** [comment]
now that the source is up on PyPI, I will rebase this on #25 once it has been merged

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+set PYFFTW_INCLUDE=%LIBRARY_INC%
+
+@rem tell pyFFTW to search for statically linked libraries
+@rem set STATIC_FFTW_DIR=%LIBRARY_LIB%
+
+@rem this variable tells pyfftw to search for fftw.lib rather than libfftw.lib
+set PYFFTW_WIN_CONDAFORGE=1
+
+@rem the following commented-out environment variable can be used to specify
+@rem a library search location for dynamic libraries
```
Comment (jjhelmus) [inline recipe/bld.bat]: The comment mentions a commented out variable, should it be updated?

File: recipe/bld.bat [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+set PYFFTW_LIB_DIR=%LIBRARY_BIN%
+
+@rem python setup.py build_ext -n -IF:\Miniconda3\conda-bld\pyfftw_1541828673207\_h_env\Library\include
```
Comment (jjhelmus) [inline recipe/bld.bat]: Does this need to be retained?

File: recipe/bld.bat [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+%PYTHON% -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```
Comment (jjhelmus) [inline recipe/bld.bat]: `--no-cache-dir` should not be used

**Comment by jjhelmus** [comment]
LGTM, a made a few minor comment but would be happy to merge as is.  Thanks for putting this together @grlee77

File: recipe/bld.bat (hunk 3 above)
Comment (grlee77) [reply recipe/bld.bat]: okay. I have removed it here and from build.sh as well.

**Comment by grlee77** [comment]
I should have put a CI skip in the prior commit. I will just cancel it manually and it should be good to go.

**Pull request merged**